src/main/java/spireMapOverhaul/zones/manasurge/powers/ManaSurgePower.java

```diff
@@ -0,0 +1,47 @@
+package spireMapOverhaul.zones.manasurge.powers;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.NonStackablePower;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractSMOPower;
+import spireMapOverhaul.zones.manasurge.ManaSurgeZone;
+
+public class ManaSurgePower extends AbstractSMOPower implements NonStackablePower {
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID("ManaSurgePower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+    public static String ZONE_ID = ManaSurgeZone.ID;
+
+    private static final float EFFECT_CHANCE = 0.5f;
+
+    public ManaSurgePower(AbstractCreature owner, int amount) {
+        super(POWER_ID,NAME,ZONE_ID, PowerType.BUFF, false, owner, amount);
```

Comment:
```suggestion
        super(POWER_ID,NAME,ZONE_ID, NeutralPowertypePatch.NEUTRAL, false, owner, amount);
```
Better to make sure this isn't affected by anything